jobs/build.jpl

```diff
@@ -79,21 +79,26 @@ node("docker" && params.NODE_LABEL) {
     K8S ctx:   ${k8s_context}""")
 
     docker_image = "${params.DOCKER_BASE}k8s:kernelci"
-    j.dockerPullWithRetry(docker_image).inside("-v $HOME/.kube:/.kube -v $HOME/.config/gcloud:/.config/gcloud -v $HOME/.azure:/.azure") {
+    j.dockerPullWithRetry(docker_image).inside("-v $HOME/.kube:/.kube-host:ro -v $HOME/.config/gcloud:/.config/gcloud -v $HOME/.azure:/.azure") {
         build_env_docker_image = j.dockerImageName(
             params.BUILD_ENVIRONMENT, params.ARCH
         )
+        env.KUBECONFIG="/tmp/.kube/config"
 
         stage("Init") {
-            /* Remove any leftover temporary files */
+            /* Remove any leftover temporary files 
             sh(script: "rm -rf *")
```

Comment:
Just remove this, leaving it make the comment meaningless. as we are not removing any leftover temporary files with the rm -rf anymowe

Reply:
done